apps/src/templates/SchoolZipSearch.jsx

```diff
@@ -54,7 +54,9 @@ export default function SchoolZipSearch({fieldNames, zip, disabled}) {
       setInputManually(true);
       sendAnalyticsEvent(EVENTS.ADD_MANUALLY_CLICKED, {});
     } else {
-      sendAnalyticsEvent(EVENTS.SCHOOL_SELECTED_FROM_LIST, {ncesId: schoolId});
+      sendAnalyticsEvent(EVENTS.SCHOOL_SELECTED_FROM_LIST, {
```

Comment:
does this need to be sent to both platforms?

Reply:
They're all going to both platforms- I abstracted the analytics call above so that I didn't have to write PLATFORMS.BOTH every time and use up a bunch more lines:)

Reply:
oo got it got it!